RELEASING.md

```diff
@@ -49,6 +49,12 @@ from these branches.
 
 We expect to cherry-pick or backport essential changes to release branches.
 
+## Release Cadence and Support
+
+AWS release new major versions of EKS approximately once per quarter (see the [EKS Kubernetes release calendar](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar)). We aim to release a stable version of `terraform-aws-eks` within one month of the upstream EKS release.
+
+We will continue to support the latest 3 major EKS versions. Small changes will be backported and released on a minor version.
```

Comment:
In SemVer language: the version number that gets bumped each quarter is the minor version.

IMO minor versions should aim to be stable, so we should only release new patch versions for:

* Security related patches
* Patches for bugs that make a release unusable

```suggestion
We will continue to support the latest 3 minor EKS versions. Security fixes will be back-ported and released on as a new patch version. Other major bugs may be back-ported based on user demand.
```

In the past I spent a lot of time back-porting too many things, and it's mostly just painful, and having a lot of change on minor versions is something we want to avoid anyway for stability... lets try to enhance on new minor versions only!